django-stubs/conf/global_settings.pyi

```diff
@@ -401,7 +403,12 @@ PASSWORD_RESET_TIMEOUT: int
 # upon login
 PASSWORD_HASHERS: list[str]
 
-AUTH_PASSWORD_VALIDATORS: list[dict[str, str]]
+@type_check_only
+class AuthPasswordValidatorsDict(TypedDict):
```

Comment:
```suggestion
class _AuthPasswordValidatorsDict(TypedDict):
```

Reply:
damn, I keep on forgetting to do that, sorry!